scripts/msct_smooth.py

```diff
@@ -270,7 +270,7 @@ def b_spline_nurbs(x, y, z, fname_centerline=None, degree=3, point_number=3000,
             PC_z = [p[2] for p in PC]
 
         import matplotlib
-        matplotlib.use('Agg')  # prevent display figure
+        # matplotlib.use('Agg')  # prevent display figure
```

Comment:
any reason for this change? (this does not seem related to the purpose of the PR)

Reply:
That was for testing, my bad again, i'll discard this change